tests/integration/test_orgs_invites.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""Integration tests for invitation methods implemented on orgs."""
+import github3
+
+from .helper import IntegrationHelper
+
+
+class TestInvitations(IntegrationHelper):
+
+    """Team integration tests."""
+
+    betamax_kwargs = {'match_requests_on': ['method', 'uri', 'json-body']}
+
+    def setUp(self):
+        super(TestInvitations, self).setUp()
+        self.basic_login()
+
+    def get_organization(self, name='mozillatw'):
+        """Get the organization for each test."""
+        self.token_login()
+        o = self.gh.organization(name)
+        assert isinstance(o, github3.orgs.Organization)
+        return o
+
+    def test_invites(self):
```

Comment:
> As a noob - it looked to me like a number of the TestOrganization cassettes relied upon a specifically crafted test org. I.e. I would not be able to get a successful run. 

That's not the case. Most presently use the same organization but they don't all have to use the same one. Once the cassette is recorded, it's fine.

> Is there a way to merge cassettes?

There's no need to merge cassettes. Each test has its own.

> For regression tests, you ask for the tests and cassettes to not be merged in. 

Can you link to this? I think I phrased it extremely poorly. What I don't want is existing tests to be appended to. I want new tests written.